src/app.js

```diff
@@ -1 +1,15 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from 'axios';
+
+export async function getAllApiData() {
+
+    //get data
+
+    try {
+
+        const response = await axios.get('https://restcountries.com/v2/all')
+        return [...response.data]
```

Comment:
Hier wordt onnodige een kopie van `response` gemaakt wat extra memory kost. Je kan de response direct teruggeven:
```suggestion
        return await axios.get('https://restcountries.com/v2/all');
```

Reply:
Klopt ik had hier aardig mee geknoeid, mijn button was een submit button die in een form stond. daardoor bleef de aanvraag van de API request altijd op pending staan en kwam ik niet verder dan dat. Dit was een workaround. 

Inmiddels ben ik natuurlijk wijzer geworden en had ik een normale button moeten gebruiken.